test/types/enum/ldelaney/enum-zippers.future

```diff
@@ -0,0 +1,2 @@
+feature request: zippered iteration of enum-types with values such as ranges
+I would expect an iterator yielding values to zip serially with value iterators
```

Comment:
Our futures tend to have a blank line in between the summary and the description, so it's easier to read.